open-api.yaml

```diff
@@ -117,6 +117,157 @@ components:
         - start
         - length
       description: Starting position and length in bytes of the matched term in the returned value
+    order:
+      type: integer
+      description: The order that this ranking rule was applied
+    customRankingRuleDetails:
+      type: object
+      properties:
+        order:
+          $ref: '#/components/schemas/order'
+        value:
+          type:
+            - string
+            - number
+            - point
+          description: The value that was used for sorting this document
+        distance:
+          type: number
+          description: The distance between the target point and the geoPoint in the document
+      required:
+        - order
+        - value
+      description: Custom rule in the form of either `attribute:direction` or `_geoPoint(lat, lng):direction`.
+    score:
+      type: number
+      description: |
+        The relevancy score of a document according to a ranking rule and relative to a search query. Higher is better.
+
+        `1.0` indicates a perfect match, `0.0` no match at all (Meilisearch should not return documents that don't match the query).
+    rankingScoreDetails:
+      type: object
+      properties:
+        words:
+          type: object
+          properties:
+            order:
+              $ref: '#/components/schemas/order'
+            matchingWords:
+              type: integer
+              description: the number of words from the query found
+            maxMatchingWords:
+              type: integer
+            score:
+              $ref: '#/components/schemas/score'
+          required:
+            - order
+            - matchingWords
+            - maxMatchingWords
+            - score
+        typo:
+          type: object
+          properties:
+            order:
+              $ref: '#/components/schemas/order'
+            typoCount:
+              type: integer
+              description: The number of typos to correct in the query to match that document.
+            maxTypoCount:
+              type: integer
+              description: The maximum number of typos that can be corrected in the query to match a document.
+            score:
+              $ref: '#/components/schemas/score'
+          required:
+            - order
+            - typoCount
+            - maxTypoCount
+            - score
+        proximity:
+          type: object
+          properties:
+            order:
+              $ref: '#/components/schemas/order'
+            score:
+              $ref: '#/components/schemas/score'
+          required:
+            - order
+            - score
+        attribute:
+          type: object
+          properties:
+            order:
+              $ref: '#/components/schemas/order'
+            attribute_ranking_order_score:
```

Comment:
@dureuill @brunoocasali this attribute should be `attributeRankingOrderScore` ?
also `query_word_distance_score` => `queryWordDistanceScore`